source/index.md

```diff
@@ -849,6 +975,17 @@ https://gateway.clearhaus.com/account
 
 ## Changes
 
+### Add payment methods
```

Comment:
We should add date of deployment here @mt-clearhaus

Reply:
:+1: